### ISSUE TITLE
Errors in the readme

### ISSUE THREAD

**Issue by Deleetdk** [stefano-meschiari/dict#1]
Interesting package. :)

There are errors in the README:

`devtools::install_github('stefano-meschiari/nicr')`

Should be `nicer`.

`mapkv(d, function(k, v) paste(key, as.character(v)))`

`key` should be `k`.

```
d2 <- dict(a=1, b=2, c=3)
mapkv_dict(d, function(k, v) v^2)
```

Should be `d2`. Otherwise it uses the other dict which has non-numeric values.

> `defaults(dict, defaults)` fill in entries from defaults into dict for any keys missing in dict.

This should be `default`, I think. `defaults` is a **plyr** function. E.g.:

```
> d
  $ color : [1] "blue"
$ pattern : [1] "solid"
  $ width : [1] 3
> default(d) = "test"
> d$nonexistent_entry
[1] "test"
```

So, the function works.

**Comment by stefano-meschiari** [comment]
Thanks for the report!

There is a `defaults` function defined, separate from the `default` function. It has the same purpose as `plyr::defaults`, but it preserves the type of the first input.

**Comment by stefano-meschiari** [comment]
Fixed in 0.5.
